warehouse/malware/tasks.py

```diff
@@ -0,0 +1,26 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import warehouse.malware.checks as checks
+
+from warehouse.tasks import task
+
+
+@task(bind=True, ignore_result=True, acks_late=True)
+def run_check(task, request, check_name, obj_id):
+    try:
+        check = getattr(checks, check_name)(request.db)
```

Comment:
It probably shouldn't happen, but we should probably watch for an `AttributeError` being raised here in the event that a `check_name` is not found for some reason.

Reply:
I agree that there needs to be more robust error handling here. Maybe we can discuss that offline.